### PR TITLE
Viewport diffuse and specular color

### PR DESCRIPTION
**Pull request by Lightye**

### CODE REVIEW

File: ui.py [hunk 1]
```diff
@@ -633,11 +633,27 @@ def draw(self, context):
         mat = context.material
         row = layout.row()
         if mat:
-            row.template_preview(context.material, show_buttons=1)
```
Comment (bsavery) [inline ui.py]: So what's the idea with this?  This removes the buttons that allow you to set the object type?

Comment (bsavery) [inline ui.py]: Those buttons work for me here (except for the last one)

**Comment by Lightye** [comment]
Yes, because there is only one object type for preview the different
buttons don't change anything.

**Comment by bsavery** [comment]
Sorry, I don't see that.... when I press the buttons I get different models.

**Comment by bsavery** [comment]
screen shot 2017-06-15 at 11 28 43 am
screen shot 2017-06-15 at 11 28 51 am
screen shot 2017-06-15 at 11 28 58 am

**Pull request merged**